articles/azure/azs-how-create-spn-portal.md

```diff
@@ -52,33 +52,31 @@ To log in and manage your resources via SPN you'll need to create an Azure appli
 
     ![Select App registrations](images/azs-browser-app-registrations.png)
```

Comment:
I'd move this to below step 4 as the screenshot is highlighting the New registration button